PluginControlInterfaceGeneric/CMakeLists.txt

```diff
@@ -17,9 +19,8 @@ OPTION (NSM_IFACE_PRESENT
 
 SET(PLUGIN_LIB_NAME PluginControlInterfaceGeneric)
 
-SET(INCLUDE_FOLDER "include")
-
-SET(CONF_ROOT "${CMAKE_INSTALL_PREFIX}/etc/controllerconf")
+set(CONTROLLER_CONFIG_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/${LIB_INSTALL_SUFFIX}/control"
```

Comment:
Hello Isogai-san,
I see, I think I got the idea on CMAKE_INSTALL_FULL_SYSCONFDIR (even if it will possibly lead to users shared configuration /usr*, while the component could rely maybe more properly on system-wide configuration, simply speaking, /etc. But I think in the end the will of having a configurable CMAKE_INSTALL_PREFIX is exactly for placing installation and configuration "somewhere else").

Also, if I got the idea behind the removal of ${LIB_INSTALL_SUFFIX} (defined and passed from AM for _installation_ purposes, not _configuration_), what do you think about introducing a e.g. LIB_CONFIG_SUFFIX in the main CMakeLists.txt, set to audiomanager by default, so that the path naming scheme is preserved?